README.md

```diff
@@ -256,6 +256,36 @@ const client = new Client(authClient);
 const result = await client.createInvoice({amount: 1000});
 ```
 
+#### Handling refresh token
+
+Access tokens do expire. If an access token is about to expire, this library will automatically use a refresh token to retrieve a fresh one. Utilising the *tokenRefresh* event is a simple approach to guarantee that you always save the most recent tokens.
+
+If token refresh fails, you can restart the OAuth Authentication flow or log the error by listening for the *tokenRefreshFailed* event.
+
+(Note: To prevent losing access to the user's token, only initialize one instance of the client per token pair at a time)
+
+```js
+const token = loadTokenForUser(); // {access_token: string, refresh_token: string, expires_at: number}
+const authClient = new auth.OAuth2User({
+  client_id: process.env.CLIENT_ID,
+  callback: "http://localhost:8080/callback",
+  scopes: ["invoices:read", "account:read", "balance:read", "invoices:create", "invoices:read", "payments:send"],
+  token: token
+});
+
+// listen to the tokenRefresh event
+authClient.on('tokenRefresh', (tokens) => {
```

Comment:
Minor: `tokenRefreshed` would be better since it has already been refreshed, and then it is consistent with `tokenRefreshFailed`.